<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdjust</class>
 <widget class="QDialog" name="DialogAdjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust Colors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="brightnessGroupBox">
     <property name="title">
      <string>Brightness</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="brightnessVLayout">
        <item>
         <widget class="QSpinBox" name="brightnessSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="brightnessHSlider">
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gammaGroupBox">
     <property name="title">
      <string>Gamma</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="gammaVLayout">
        <item>
         <widget class="QDoubleSpinBox" name="gammaDoubleSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>7.990000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="gammaHSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>799</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="constrastGroupBox">
     <property name="title">
      <string>Contrast</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="contrastVLayout">
        <item>
         <widget class="QSpinBox" name="contrastSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>-127</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="contrastHSlider">
          <property name="minimum">
           <number>-127</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="channeslGroupBox">
     <property name="title">
      <string>Apply to Channels:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="redCheckBox">
        <property name="text">
         <string>Red</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="greenCheckBox">
        <property name="text">
         <string>Green</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="blueCheckBox">
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="alphaCheckBox">
        <property name="text">
         <string>Alpha</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAdjust</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAdjust</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
